config/internal/base/deployment.yaml.tmpl

```diff
@@ -60,6 +60,9 @@ spec:
             - name: MM_SERVICE_NAME
               value: {{.ServiceName}}
               # External gRPC port of the service, should match ports.containerPort
+            - name: MM_PAYLOAD_PROCESSORS
+              value: {{.PayloadProcessors}}
```

Comment:
Since this sometimes may be empty, might be best to surround the whole name/value with an `{{if ...` like what's done for `MM_METRICS` below.

Alternatively we could add it as a conditional step in the [addMMEnvVars](https://github.com/kserve/modelmesh-serving/blob/main/controllers/modelmesh/modelmesh.go#L215) function.

I'm not sure to be honest why some of the env vars get set via the template and others programmatically, I think it's just how the code evolved.